casper-helpers.js

```diff
@@ -0,0 +1,323 @@
+var system = require('system');
```

Comment:
Not sure if this belongs here but it's a good step. I'd put this and the ```casper-shim``` in a ```marketplace-gulp/tests``` folder